docs/components/PaginationButtonsDocs.js

```diff
@@ -50,6 +51,10 @@ class PaginationButtonsDocs extends React.Component {
         <h5>totalPages <label>Number</label></h5>
         <p><em>(required)</em> The total number of pages.</p>
 
+        <h5>type <label>String</label></h5>
+        <p>Default: <Code>'primary'</Code></p>
+        <p>The sets the button type.</p>
```

Comment:
weird grammar here.

Reply:
good catch, I never ended up finishing that one. just updated